### DISCUSSION TITLE
How many entries should we have for v1?

### DISCUSSION THREAD

**Discussion by JasonMorgan** [cncf/glossary#3] [General]
I think we should set a target of like 50 entries for our initial release. Anyone have any thoughts?

**Comment by caniszczyk** [comment]
I think 25-50 is a fine number to start, keep it simple to hit v1.0 :)

On Wed, Feb 3, 2021 at 11:47 AM Jason Morgan <notifications@github.com>
wrote:

> I think we should set a target of like 50 entries for our initial release.
> Anyone have any thoughts?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cncf/glossary/discussions/3>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPSIOOMGPFP7NH45FVHJDS5GDZRANCNFSM4XBI4YZQ>
> .
>


-- 
Cheers,

Chris Aniszczyk
http://aniszczyk.org

**Comment by CathPag** [comment]
@JasonMorgan @IdealUsrname @caniszczyk [Here's a list ](https://docs.google.com/spreadsheets/d/1QvGbINXH60kOibr-Yf5IKHGf2aX2a7cTzQQu_aqJz58/edit#gid=0&fvid=1364988504)with all terms and a proposition for 25 that should go into V1. What do you think?

**Reply by IdealUsrname** [reply]
I see no issues with these suggestions for v1. :)

**Comment by CathPag** [comment]
If we stick with those terms, our "homework" for next week is getting API gateway, API, and auto scaling ready for V1 by the end of next week.

**Reply by CathPag** [reply]
@JasonMorgan @IdealUsrname, not sure why I added only three terms even though I said we need to get 4 done per week. Anyhow, our homework for next week is:
- API gateway
- API
- Autoscaling
- Blue green deployment

**Comment by CathPag** [comment]
[Google doc ](https://docs.google.com/document/d/1bAgNu1GkreUdFCFzBjCwwgqQ15xz49-eiAQ58QCYgmI/edit)for the terms of the week.
